pkg/sqlcmd/sqlcmd.go

```diff
@@ -284,7 +287,12 @@ func (s *Sqlcmd) IncludeFile(path string, processAll bool) error {
 	}
 	defer f.Close()
 	b := s.batch.batchline
-	scanner := bufio.NewScanner(f)
+	win16be := unicode.UTF16(unicode.BigEndian, unicode.IgnoreBOM)

```

Comment:
Please add an entry to the readme describing the supported text encodings of the files.  Having test files of each type will be great!

  The ODBC sqlcmd claims 
`sqlcmd automatically recognizes both big-endian and little-endian Unicode input files. If the -u option has been specified, the output will always be little-endian Unicode.`

We haven't implemented `-u` for output yet.

Reply:
aha... so need to add an option... I will add the tests today.

Reply:
if you are fine with UTF8 output you can skip implementing `-u` for now to unblock your input needs.